examples/apps/ai-collab/tsconfig.json

```diff
@@ -20,6 +20,7 @@
 		"paths": {
 			"@/*": ["./src/*"],
 		},
+		"allowJs": true,
```

Comment:
I also had this automatically applied while working on my ai-collab example and discarded the change. Is this relevant to the changes that you are making?

Reply:
Yeah, a weird merge might have made this come back, it should be removed.